app/Resources/views/about/board_and_team.html.twig

```diff
@@ -26,14 +26,19 @@
             <table>
                 {% for department in departments %}
                     <tr>
-                        <th colspan="2">
+                        <th colspan="3">
                             <h3>{{ department.shortName }}</h3>
                         </th>
                     </tr>
                     {% for team in department.teams %}
                         <tr>
                             <td><h6><a href="{{ path('team_page', {'id': team.id}) }}">{{ team.name }}</a></h6></td>
                             <td><span class="team-short-description">{{ team.shortDescription }}</span></td>
+                            {% if team.acceptApplication %}
+                                <td> <a href="{{ path('team_application',{id:team.id}) }}" class="button tiny"> Søk</a></td>
```

Comment:
Lagde egen branch for den, som e jobba parallelt med. Tru e gjor nå dumt derimot, så endringa vart med her i tillegg. Kan lag egen PR med den andre branchen, men tenkt d va unødvendig. 
Så ja, den kan godt vårrå med her.